maintainers/maintainer-list.nix

```diff
@@ -13172,4 +13172,10 @@
     github = "sei40kr";
     githubId = 11665236;
   };
+  shikanime = {
```

Comment:
Adding a new entry to `maintainer-list.nix ` should be a separate commit in this PR: